### ISSUE TITLE
Display Groups

### ISSUE THREAD

**Issue by lukasepple** [hannesm/jackline#84]
Like in mcabber's UI?

**Comment by hannesm** [comment]
not very high on my priority list.. but might be easy after mucng is there..

**Comment by hannesm** [comment]
the fundamental problem in groups I can see for the jackine interface is: a jid might be member of any number of groups; which one to highlight for notifications?

**Comment by lukasepple** [comment]
Yeah, this is a difficult question, mcabber for example does only display the JID in one group I think.
